### PR TITLE
Test dangerfile in test and development environment

### PR DESCRIPTION
**Pull request by oliverguenther**
Adds a very basic dangerfile to test for: 

- Brakeman issues in the modified / added files (used with a plugin, doesn't require any entry in the dangerfile)
- Testing for jasmine focus keywords (fit, describe)

### CODE REVIEW

File: app/models/journal/aggregated_journal.rb [hunk 1]
```diff
@@ -56,7 +56,7 @@ def with_notes_id(notes_id)
       # Therefore we have to provide the notes_id to the aggregation on top of it being used
       # in the where clause to pick the desired AggregatedJournal.
       raw_journal = query_aggregated_journals(journal_id: notes_id)
-                    .where("#{table_name}.id = ?", notes_id)
+                    .where("#{table_name}.id = #{notes_id}")
```
Comment (openprojectci) [inline app/models/journal/aggregated_journal.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/023ed259d6890e21641dfb6781cc80185f02a559/app/models/journal/aggregated_journal.rb#L59"></span>[brakeman] Possible SQL injection</td>
    </tr>
  </tbody>
</table>

**Comment by openprojectci** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">jasmine fdescribe left in tests</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by ulferts — APPROVED** [review]
Looking mighty fine. Thanks.

**Review by machisuji — APPROVED** [review]
Neat. :+1:

**Pull request merged**